Making columns of DataFrame rdered

SteveNY-Tibco opened the pull request:
Since sort operation sorts both by column (axis = 0) and by row (axis = 1), we need a way to represent the order of row. This modification only for making DataFrame keep the order of its columns.

SteveNY-Tibco committed:
Making columns of DataFrame rdered

Since sort operation sorts both by column (axis = 0) and by row (axis = 1), we need a way to represent the order of row. This modification only for making DataFrame keep the order of its columns.

abramvandergeest commented:
I think my biggest response to this is that I have used the dataframe struct in other ops that need to be updated to work with .data

SteveNY-Tibco committed:
Adding sort operation

SteveNY-Tibco committed:
Add for sort operation

SteveNY-Tibco committed:
changes about DataFrame and sort operation

1. Finalize sort operation.
2. Enhance DataFrame, DataFrameSorter, SortableTuple
3. modifies operation according to the change of DataFrame

SteveNY-Tibco committed:
Fix some bugs and better test case
